lnbits/core/templates/core/index.html

```diff
@@ -160,15 +160,19 @@
                     class="text-center text-grey-6"
                     v-if="authAction !== 'reset'"
                   >
-                    <p v-if="authAction === 'login'" class="q-mb-none">
+                    <p
+                      v-if="authAction === 'login' && {{ LNBITS_NEW_ACCOUNTS_ALLOWED | tojson }}"
```

Comment:
can you introduce a this? 
```suggestion
                      v-if="authAction === 'login' && allowed_new_users"
```